library/dialog.js

```diff
@@ -208,8 +208,11 @@ function dlgopen(url, winname, width, height, forceNewWindow, title, opts) {
     jQuery(function () {
         // Check for dependencies we will need.
         // webroot_url is a global defined in main_screen.php or main.php.
-
-        let bscss = top.webroot_url + '/public/assets/bootstrap-3-3-4/dist/css/bootstrap.min.css';
+        let bscss = "";
+        if (top.jsLanguageDirection !== 'rtl')
+            bscss = top.webroot_url + '/public/assets/bootstrap-3-3-4/dist/css/bootstrap.min.css';
+        else
+            bscss = top.webroot_url + '/public/assets/bootstrap-rtl-3-3-4/dist/css/bootstrap-rtl.min.css';
```

Comment:
hi @sjpadgett 
Logic should be:
1. always include bootstrap.min.css
2. if rtl, then also include bootstrap-rtl.min.css (ordered after bootstrap.min.css is included)

Reply:
I see. Will fix and go ahead and bring in. Thanks.